lib/farside/deadcheck.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Farside.Server.DeadCheck do
+  @moduledoc """
+  Module to check/validate the instance list only for servers with empty instance list every 90 secs, if a sync/check process isnt already running
```

Comment:
```suggestion
  Module to check/validate the instance list only for servers with empty instance list once a day, if a sync/check process isnt already running
```

(unless you actually meant to change `86_400_000` below to 90 seconds)

Reply:
no its supposed to be every 24h i changed it to every 12, as each instance times out 50x it will be put  in dead status which then is rechecked every 12 hrs